src/progressbar/progressbar.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+angular.module('mgcrea.ngStrap.progressbar', [])
+.provider('$progressbar', function (){
+    var defaults = {
+      barType: '',
+      animate: function (){ return true;}
+    };
+
+    this.$get = function (){
+      return {
+        defaults: defaults
+      };
+    };
+  })
+  .directive('bsProgressbar', function ($progressbar){
+    return {
+      restrict: 'E',
+      transclude: true,
+      replace: true,
+      templateUrl: 'progressbar/progressbar.tpl.html',
+      scope:{
```

Comment:
In the past, the isolated scopes were buggy, and leaked to other directives (see angular/angular.js/issues/1924). So AngularStrap avoided them since they were mostly decorator directives and this bug broke ngModel bindings.
It is now fixed but the project has not migrated yet, for the sake of consistency, I'd rather keep defaulting to scope:true (leaving the isolated scope definition in comments to ease a future update). At some point (maybe for the next minor), we should migrate all directives at once.